pandas/tseries/common.py

```diff
@@ -106,6 +106,8 @@ def _delegate_property_get(self, name):
         elif not is_list_like(result):
             return result
 
+        result = np.asarray(result)
+
```

Comment:
Another reason to convert to an array is so that `Series` does not take a copy of the values (which it does if you pass an Index object I think)